Specification.md

```diff
@@ -110,25 +111,34 @@ A *range* is an *entry* that represents the time span between two points in time
 
 It MUST consist of two values that denote the start and the end.
 Start and end MUST be written in chronological order.
+They MAY be equal.
 
 There MUST be a `-` between the two values.
 There MAY appear “spaces” on either side of the `-`;
 for this case it is RECOMMENDED to use exactly one “space” on both sides.
 
-The start value MUST be a *time*.
-It MAY be prefixed with a `<` to indicate that
-this *time* is referring to the day before the *record’s* date,
-e.g. `<23:00`.
-
-The end value MUST be either a *time* or a placeholder for a *time*.
-- In case the end value is a *time* it MAY be suffixed with a `>` to indicate
-  that this *time* is referring to the day after the *record’s* date,
-  e.g. `0:30>`.
-- In case the end value is a placeholder the *range* is considered to be *open-ended*,
-  which means that the end *time* is not determined yet.
-  The placeholder MUST be denoted by a `?`, e.g. `9:00 - ?`;
-  the `?` MAY be repeated, e.g. `9:00 - ???`.
-  An *open-ended range* MUST NOT occur more than once per record.
+*Time* values MAY be *shifted* to the next or to the previous day:
+- To associate the *time* with the day before the *record’s* date,
+  a `<` prefix MUST be used,
+  e.g. `<23:00`.
+- To associate the *time* with the day after the *record’s* date,
+  a `>` suffix MUST be used,
+  e.g. `1:30>`.
+
+Examples of *ranges* with *shifted times*:
+`<23:00 - 5:00`, `19:00 - 0:30>`, `<22:25 - <23:59`, `<15:00 - 12:00>`.
```

Comment:
Very minor detail; in these examples, all uses for shifting the time are provided *except* the case for a start time the next day (which implies specifying two times in the next day).

Reply:
I actually didn’t want to go overboard with the examples, but not that you point it out I see that one can stumble across this. So I agree that showing off possible combination makes sense here. (https://github.com/jotaen/klog/pull/88/commits/57ddb624d6afa480fbc5e72b304486458cc46e58)